### DISCUSSION TITLE
Groq is not accessible

### DISCUSSION THREAD

**Discussion by legaltextai** [open-webui/open-webui#4609] [General]
![Screenshot 2024-08-14 9 16 41 PM](https://github.com/user-attachments/assets/0245bd28-58ce-4bdd-a8c8-ea84efe4c8bd)
Tried to add via connections and then via pipeline. The same error. 
Thank  you

**Comment by justinh-rahb** [comment]
Working fine in my test environments.

**Comment by legaltextai** [comment]
I am on linux server. Other models work fine.

**Comment by legaltextai** [comment]
Anthropic is not accessible too. Added via a pipeline

**Comment by mxmp210** [comment]
Some providers do not support certain options, so check with the runtime options with the completion request. 

100% API compatibility is still missing from a lot of vendors, which results in 400 errors.

**Reply by mxmp210** [reply]
Network error is different thing, that is when server is not able to connect to external service or there is a transport error, check with network settings, firewall, dns, and timeouts.
